protocol-configuration/mainnetAddresses.ts

```diff
@@ -311,6 +311,26 @@ const MainnetAddresses: MainnetAddresses = {
     address: '0x374628EBE7Ef6AcA0574e750B618097531A26Ff8',
     category: AddressCategory.PCV
   },
+  balancerFeiWethPool: {
```

Comment:
Keep this contract clean by moving addresses to their respective locations.

We can make a tool that automates this sorting maybe?